integration_tests/test_list_09.py

```diff
@@ -0,0 +1,38 @@
+from ltypes import i32
+
+def test_list_concat():
+    x: list[i32] = []
+    y: list[i32] = []
+    z: list[i32] = x + y
```

Comment:
`print(z)` throws `LCompilersException: Printing support is not available for list type.`

Reply:
If I replace it with
```py
for i in range(len(z)):
	print(z[i])
```
it throws: `LCompilersException: visit_DoLoop() not implemented`

Reply:
Sorry for the confusion. I didn't use the `print_list` and `do_loops` passes. After passing the passes it works.